src/icecreammonitor.cc

```diff
@@ -86,11 +86,18 @@ void IcecreamMonitor::checkScheduler(bool deleteit)
 void IcecreamMonitor::registerNotify(int fd, QSocketNotifier::Type type, const char *slot)
 {
     if (m_fd_notify) {
+        if(m_fd_notify->socket() != fd || m_fd_notify->type() != type) {
```

Comment:
missing space

Reply:
And you tell me about funny comments?

Reply:
anything unclear here?

Reply:
Unless you're the new Icemon maintainer, no. I'm the guy who does the work and so I get to decide that caring about some irrelevant space is a waste of time.

Reply:
and that is how exactly the same thing as a commit message that claims something that is literally not possible?

Reply:
Look, if you've got time to waste on this, why don't you actually do something useful instead of writing cryptic complaints?

Reply:
your priorities with all due respect, but don't you think that you can do better than trying to ridicule me?

Reply:
If you want to see it that way, that's your choice. I'm not going to spend more time on this.